### DISCUSSION TITLE
[기능 제안] 최근 검색어를 캐싱하는 기능이 있으면 좋겠어요.

### DISCUSSION THREAD

**Discussion by lgvv** [KU-Stacks/kuring-lite-ios#5] [Ideas]
특정 검색어를 자주 검색하는데, 계속 다시 입력하기가 귀찮아요.
검색어를 저장하는 로직이 있으면 얼마나 좋을까요!

**Comment by x-0o0** [comment]
👋 안녕하세요. 쿠링팀의 프로젝트 매니저 재성입니다.

우선, 쿠링의 성장에 큰 도움이 되는 좋은 기능을 제안해주셔서 감사합니다. 제안해주신 기능이 쿠링 서비스에 **매우 필요할 것**이라는 판단이 들어 가능한한 빨리 개발을 시작할 예정입니다. 물론 직접 PR을 생성하실 수도 있습니다.

개발이 시작되거나 PR이 시작될 때 추가적인 커멘트가 생성될 예정입니다.

추가적인 이슈가 있다면 새로운 이슈를 리포트 하거나 여기에 커멘트를 남겨주세요.

---
👋 Ola, this is Jaesung, who's PM of Kuring team.

First of all, thanks for your wonderful feature suggestion. Our iOS development team will work on it *immediately*. You can also create your own pull request.

There will be additional comment when it starts to develop the feature or when you create a new PR regarding the issue.

If you have any further issue, please report a new issue or leave comment here.

**Comment by x-0o0** [comment]
[PR link](https://github.com/KU-Stacks/kuring-lite-ios/pull/3)
